jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -23,6 +23,10 @@
 
 package openj9.tools.attach.diagnostics.base;
 
+import static com.ibm.tools.attach.target.IPC.logMessage;
+import static openj9.tools.attach.diagnostics.base.DiagnosticProperties.makeErrorProperties;
+import static openj9.tools.attach.diagnostics.base.DiagnosticProperties.makeStringResult;
```

Comment:
Please don't use static imports like this: it only makes the code harder to understand.